packages/models/user_profiles.coffee

```diff
@@ -0,0 +1,24 @@
+UserProfiles = new Mongo.Collection('userProfile')
+UserProfile = Astro.Class
+  name: 'UserProfile'
+  collection: UserProfiles
+  transform: true
+  fields: 
+    fullName: 'string'
+    jobTitle: 'string'
+    bio: 'string'
+    emailHidden: 'boolean'
+    userId: 'string'
+
+  methods:
+    update: (fields, callback) ->
+      this.set(fields)
+      this.save ->
+        callback?()
+
+if Meteor.isServer
```

Comment:
@aslagle how do you feel about this being in this file?  I put it here since it defines the rules for creating UserProfiles but it's not actually related to the model definition.